docs/overlay.md

```diff
@@ -33,6 +33,39 @@ const OverlayExample = () => {
 };
 ```
 
+> Web-platform specific note:
+>
+> You **must** pass a valid React Native [`Modal`](https://facebook.github.io/react-native/docs/modal) component implementation
+> into [`ModalComponent`](#modalcomponent) prop because `Modal` component is not implemented yet in `react-native-web`
+
+```jsx
+import React, { useState } from 'react';
+import { Button, Overlay } from 'react-native-elements';
```

Comment:
Since this is the same example as above, I think we can just put the lines of the import and where ModalComponent is used

Reply:
@iRoachie this action can confuse those who do not use web-platform... I will make shorter example...

Reply:
@iRoachie please look at it now. I don't know how to do it better